File: PHP/Easy/Filters/Array/2185. Counting Words With a Given Prefix.php
<?php
// https://leetcode.com/problems/counting-words-with-a-given-prefix/description/
function prefixCount($words, $pref)
{
    $count = 0;
    foreach ($words as $word) {
        if (substr($word, 0, strlen($pref)) == $pref) {
            $count++;
        }

    }
    return $count;

}

$words = ["pay", "attention", "practice", "attend"];
$pref = "at";
echo prefixCount($words, $pref);

File: Python/Easy/Filter/String/2185. Counting Words With a Given Prefix.py
"""
https://leetcode.com/problems/counting-words-with-a-given-prefix/description/
"""


def prefixCount(words: list[str], pref: str) -> int:
    count = 0
    for word in words:
        if word.startswith(pref):
            count += 1
    return count


words = ["pay","attention","practice","attend"]
pref = "at"

print(prefixCount(words, pref))
